rust/gui-client/src-tauri/src/client/gui/system_tray.rs

```diff
@@ -311,10 +342,12 @@ mod tests {
     fn no_resources_no_favorites() {
         let resources = vec![];
         let favorites = Default::default();
+        let disabled_resources = Default::default();
         let input = AppState::SignedIn(SignedIn {
             actor_name: "Jane Doe",
             favorite_resources: &favorites,
             resources: &resources,
+            disabled_resources: &disabled_resources,
         });
```

Comment:
A fair bit of duplication here with constructing `SignedIn`. Maybe another helper fn could make this a bit more concise?